src/Components/TreeView/TreeView.js

```diff
@@ -178,106 +178,110 @@ const File = ({
-												onModal();
-											} else {
-												const a = document.createElement('a');
-												document.body.appendChild(a);
-												const url = window.URL.createObjectURL(data.res);
-												a.href = url;
-												a.download = name;
-												a.target = '_blank';
-												a.click();
-												setTimeout(() => {
-													onModal();
-													window.URL.revokeObjectURL(url);
-													document.body.removeChild(a);
-												}, 0);
-											}
-										});
-									}}
-									width={40}
-									height={40}
-									alt="get an object by link"
-								/>
-								<img
-									src="/img/icons/manage/unlock.png"
-									className="manage_icon"
-									onClick={() => {
-										onModal('loading');
-										api('GET', `/get/${containerItem.containerId}/${objectItem.address.objectId}`, {}, {
-											[ContentTypeHeader]: "application/json",
-											[AuthorizationHeader]: `Bearer ${walletData.tokens.object.bearer}`,
-										}).then((data) => {
-											if (data.status !== 200) {
-												onModal('failed', 'Something went wrong, try again');
-											} else if (data.header.indexOf("image/") !== -1 || data.header === 'text/plain; charset=utf-8') {
-												const fileURL = URL.createObjectURL(data.res);
-												window.open(fileURL, '_blank');
-												onModal();
-											} else {
-												const a = document.createElement('a');
-												document.body.appendChild(a);
-												const url = window.URL.createObjectURL(data.res);
-												a.href = url;
-												a.download = name;
-												a.target = '_blank';
-												a.click();
-												setTimeout(() => {
-													onModal();
-													window.URL.revokeObjectURL(url);
-													document.body.removeChild(a);
-												}, 0);
-											}
-										});
-									}}
-									width={40}
-									height={40}
-									alt="get an object by bearer"
-								/>
-								<img
-									src="/img/icons/manage/download.png"
-									className="manage_icon"
-									onClick={() => {
-										onModal('loading');
-										api('GET', `/get/${containerItem.containerId}/${objectItem.address.objectId}`, {}, {
-											[ContentTypeHeader]: "application/json",
-											[AuthorizationHeader]: `Bearer ${walletData.tokens.object.bearer}`,
-										}).then((data) => {
-											const a = document.createElement('a');
-											document.body.appendChild(a);
-											const url = window.URL.createObjectURL(data.res);
-											a.href = url;
-											a.download = name;
-											a.target = '_blank';
-											a.click();
-											setTimeout(() => {
-												onModal();
-												window.URL.revokeObjectURL(url);
-												document.body.removeChild(a);
-											}, 0);
-										});
-									}}
-									width={40}
-									height={40}
-									alt="download an object"
-								/>
+								{!window.OneGate && (
```

Comment:
The only thing for OneGate is `delete`? Downloads (with appropriate Content-Disposition) get blocked too?